go.mod

```diff
@@ -1,14 +1,10 @@
 module github.com/openfaas-incubator/nats-connector
 
+go 1.13
```

Comment:
Please remove the Go modules and use `dep` instead for consistency with the rest of OpenFaaS.

Reply:
You may have found Go modules before, but this was never a completed effort by @johnmccabe

Reply:
Sorry to be rude, but I don't have time to do this.

Reply:
It's two commands:

dep init
git commit

Reply:
Given that you feel it's unreasonable to ask for this tiny simple change, I'll merge and find someone else to sort it out.

Reply:
Just bear in mind that OSS works two ways, we also don't have time to entertain every user request, issue and PR, but are flexible and understand it's give and take.